migrate.js

```diff
@@ -259,32 +289,23 @@ exports.init = function init(sbot, config) {
         if (err) return console.error(err)
 
         // Inform the other parts of ssb-db2 that migration is done
-        clearInterval(progressInterval)
-        emitProgressEvent()
-        synchronized.set(true)
-        debug('done migrating %s msgs from old log', msgCountOldLog)
-        drainAborter = null
-
-        // Setup periodic `debug` reporter of live msgs migrated
-        let liveMsgCount = 0
-        if (debug.enabled) {
-          setInterval(() => {
-            if (liveMsgCount === 0) return
-            debug('%d msgs synced from old log to new log', liveMsgCount)
-            liveMsgCount = 0
-          }, 2000).unref()
+        function doneMigrating() {
+          clearInterval(progressInterval)
+          emitProgressEvent()
+          synchronized.set(true)
+          debug('done migrating %s msgs from old log', msgCountOldLog)
+          drainAborter = null
         }
 
-        // Setup migration of live new msgs identified on the old log
-        oldLog.newMsgObv((msg) => {
-          const guard = guardAgainstDecryptedMsg(msg)
-          if (guard) throw guard
-
-          writeToNewLog(toBIPF(msg), () => {
-            liveMsgCount++
-            if (liveMsgCount % 100 === 0) emitProgressEvent()
+        if (config.db2.dangerouslyKillFlumeWhenMigrated) {
```

Comment:
One thing that I wanted to do here was `oldLog.close` but I noticed such method does not exist. :eyes: I think we're safe enough from the possibility of FlumelogOffset instance being used, but ideally it would be good to nuke that possibility.